src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './Calculator.css';
+
+const Calculator = () => (
+  <div className="main">
+    <div className="calculator">
+      <div className="screen">
+        <input type="text" value="0" className="silver" />
+      </div>
+      <div className="row first-row">
+        <input value="AC" type="button" />
+        <input value="+/-" type="button" />
+        <input value="%" type="button" />
+        <input value="/" type="button" className="color" />
+      </div>
+      <div className="row second-row">
+        <input value="7" type="button" />
+        <input value="8" type="button" />
+        <input value="9" type="button" />
+        <input value="X" type="button" className="color" />
+      </div>
+      <div className="row third-row">
+        <input value="4" type="button" />
+        <input value="5" type="button" />
+        <input value="6" type="button" />
+        <input value="-" type="button" className="color" />
+      </div>
+      <div className="row fourth-row">
+        <input value="1" type="button" />
+        <input value="2" type="button" />
+        <input value="3" type="button" />
+        <input value="+" type="button" className="color" />
+      </div>
+      <div className="row fifth-row">
+        <input value="0" type="button" className="expand" />
+        <input value="." type="button" />
+        <input value="=" type="button" className="color" />
+      </div>
+    </div>
+  </div>
```

Comment:
As part of the project requirements, the Calculator.js component should have at least one child component. Kindly update your project and add at least one child component to the Calculator.js component to match this requirement. :pray:

Reply:
@jerryowusu Can you please explain this to me, I have a hard time understanding what you suggest I should implement.